React/React.csproj

```diff
@@ -10,6 +10,7 @@
     <PackageReference Include="JavaScriptEngineSwitcher.Extensions.MsDependencyInjection" Version="3.19.0" />
     <PackageReference Include="JavaScriptEngineSwitcher.V8" Version="3.20.8" />
     <PackageReference Include="JavaScriptEngineSwitcher.V8.Native.win-x64" Version="3.20.8" />
+    <PackageReference Include="Microsoft.ApplicationInsights" Version="2.20.0" />
```

Comment:
We cannot depend on ApplicationInsights in library and force end user to refrence it. Let's depend on some abstraction. Maybye we can allow to write and register Decroator?

Reply:
agreed to remove dependency and inject implementation from client code - done